aep/general/0146/aep.md.j2

```diff
@@ -39,11 +34,51 @@ when there is a large (or unlimited) number of potential options, or when there
 is a large resource structure that would require a long series of "cascading
 oneofs".
 
-**Note:** Adding additional possible fields to an existing `oneof` is a
-non-breaking change, but moving existing fields into or out of a `oneof` is
-breaking (it creates a backwards-incompatible change in Go protobuf stubs).
+**Note:** Adding additional fields or schemas to an existing `oneof` is a
+non-breaking change. However, removing or moving fields is breaking.
 
-#### Maps
+{% tab proto %}
+
+Because the individual fields in the `oneof` have different keys, a developer
+can programmatically determine which (if any) of the fields is populated.
+
+Moving existing fields into or out of a `oneof` creates a
+backwards-incompatible change in Go protobuf stubs.
+
+{% tab oas %}
+
+```yaml
+components:
+  schemas:
+    PaymentMethod:
+      type: object
+      properties:
+        payment:
+          oneOf:
```

Comment:
the pattern here outlines one specific way to use `oneOf`. Another one is to support multiple top-level inline schemas, although one could interpret this section to apply to that too.

Is it worth showing a top-level example? or is this clear enough?

Reply:
I think it's clear enough, but I'll let you override.